plugins/aggregators/histogram/histogram.go

```diff
@@ -66,7 +68,8 @@ type groupedByCountFields struct {
 // NewHistogramAggregator creates new histogram aggregator
 func NewHistogramAggregator() *HistogramAggregator {
 	h := &HistogramAggregator{
-		Cumulative: true,
+		Cumulative:       true,
+		PushOnlyOnUpdate: false,
```

Comment:
This doesn't to be set as it is the Golang default, so please remove the `PushOnlyOnUpdate: false,` line.